arcgis-ios-sdk-samples/Features/Feature layer rendering mode (map)/FeatureLayerRenderingModeMap.storyboard

```diff
@@ -13,37 +14,38 @@
             <objects>
                 <viewController storyboardIdentifier="FeatureLayerRenderingModeMapStoryboard" title="Feature layer rendering mode (map)" useStoryboardIdentifierAsRestorationIdentifier="YES" id="7OF-WW-ois" customClass="FeatureLayerRenderingModeMapViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
                     <view key="view" contentMode="scaleToFill" id="tmZ-VH-cn3">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="812"/>
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="MHI-Dx-Wlm" userLabel="Dynamic Map View" customClass="AGSMapView">
```

Comment:
![image](https://user-images.githubusercontent.com/9660181/99710153-0c302280-2a55-11eb-85f6-c3f24fd2e798.png)

The views have wrong constraints that are not snapping to the superview.